src/main/python/TravelTimeReporter.py

```diff
@@ -106,6 +106,7 @@ def read_active_skims(self):
         """
         Reads active skims into memory as data frames
         """
+        # Read MAZ-level skims
```

Comment:
I love these kinds of comments. We need to do more of this throughout our codebase.